### PR TITLE
Update update-daemon-set.md

### PR DESCRIPTION
**Pull request by MdSahil-oss**
<h3>What type of PR is this ?</h3>
Improve documentation.

<h3>Which type of issue this PR fixes</h3>
fixes 31311(issue) and replaces 31354.

### CODE REVIEW

File: content/en/docs/tasks/manage-daemon/update-daemon-set.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ You may want to set
 (default to 1),
 [`.spec.minReadySeconds`](/docs/concepts/workloads/controllers/deployment/#min-ready-seconds) 
 (default to 0) and 
-[`.spec.maxSurge`](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#max-surge) 
+[`.spec.updateStrategy.rollingUpdate.maxSurge`](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/#DaemonSetSpec)
```
Comment (sftim) [inline content/en/docs/tasks/manage-daemon/update-daemon-set.md:42]:
```suggestion
[`.spec.updateStrategy.rollingUpdate.maxSurge`](/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/#DaemonSetSpec)
```

**Review by sftim — COMMENTED** [review]
LGTM

Would be even better with 1 more tweak - see inline feedback

File: content/en/docs/tasks/manage-daemon/update-daemon-set.md (hunk 1 above)
Comment (MdSahil-oss) [reply content/en/docs/tasks/manage-daemon/update-daemon-set.md]: @sftim I have made changes as per your suggestion

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**